.github/workflows/peer-rec-CI.yml

```diff
@@ -0,0 +1,44 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the action will run. 
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master, hosted-docs ]
+  pull_request:
+    branches: [ master, hosted-docs ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
```

Comment:
more of a question - are these steps running in sequence or in parallel? Either is fine for now but eventually the building/deployment and documenting scripts should be ran concurrently as part of our CD script on a push to master. Our CI script would run on every pull request and would also run concurrently. The jobs/checks for a pull request include developer approval, passing tests, and a successful build.

Reply:
They're running in sequence. I think we want to generate new docs only if tests/builds pass.